app/src/main/java/ar/com/wolox/android/example/ui/login/LoginPresenter.java

```diff
@@ -0,0 +1,88 @@
+package ar.com.wolox.android.example.ui.login;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.widget.EditText;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.inject.Inject;
+import ar.com.wolox.wolmo.core.presenter.BasePresenter;
+
+/**
+ * LoginPresenter
+ */
+
+public class LoginPresenter extends BasePresenter<ILoginView> {
+
+    final String emailReq = "Email is required";
+    final String passReq = "Password is required";
+    final String invEmail = "Invalid Email";
+    final String emailKey = "ar.com.wolox.android.example.emailCredential";
```

Comment:
Let's make it (and `passKey`) a Java constant. The way we have this is like: `final static String`. Also, the naming convention for constants is with UPPER_CASE.